LICENSE.txt

```diff
@@ -1,27 +0,0 @@
-Copyright (c) 2015-2023, conda-forge
```

Comment:
Please don't remove our license!

Reply:
I copied it back in.  My git-foo isn't good enough to figure out how to "never have removed it."  Want me to squash this PR into one commit?

Reply:
Sure. That works.